tools/test_value_scraper_vmc.py

```diff
@@ -102,6 +102,31 @@ def get_vcenter_server_detail(args):
     return output["vcenter_detail"]
 
 
+def get_vcenter_admin_detail(args):
```

Comment:
Wow!

I'm surprised passwords can be read (and not in a good way).

Reply:
yes, we can get the required credentials from the response of vmc_sddc.get_by_id

Reply:
Something that we should probably consider doing is adding an `unmask_passwords` parameter, or something similar, and default in all `ret` structures to replacing the passwords with just `<REDACTED>` or something like that.

FWIW, it *is* useful because a map jinja file can use Salt to get the credentials for other states, but... :shrug: